benchmarks/Dockerfile

```diff
@@ -0,0 +1,128 @@
+RUN git clone https://github.com/tldr-pages/tldr-c-client.git \
+        && cd tldr-c-client \
+        && DEBIAN_FRONTEND=noninteractive ./deps.sh \
+        && make \
+        && mkdir /build-outputs /deps \
+        && cp tldr /build-outputs/tldr-c \
+        && cp deps.sh /deps/tldr-c-deps.sh
+
+################################################################################
+
+FROM haskell:8.6 AS haskell-builder
+
+WORKDIR /build
+
+RUN git clone https://github.com/psibi/tldr-hs.git \
+        && cd tldr-hs \
+        && stack build
+
+RUN git clone https://github.com/gutjuri/fast-tldr \
+        && cd fast-tldr \
+        && stack build
+
+RUN mkdir /build-outputs \
+        && find tldr-hs/.stack-work/dist -type f -iname tldr -exec mv '{}' /build-outputs/tldr-hs \; \
+        && find fast-tldr/.stack-work/dist -type f -iname tldr -exec mv '{}' /build-outputs/fast-tldr \;
+
+################################################################################
+
+FROM node:slim AS node-builder
+
+WORKDIR /build-outputs
+RUN npm install tldr \
+        && cp $(which node) . \
+        && echo './node -- ./node_modules/.bin/tldr "$@"' > tldr-node \
+        && chmod +x tldr-node
+
+################################################################################
+
+FROM euantorano/zig:master AS zig-builder
+
+WORKDIR /build
+RUN apk add git \
+        && git clone https://gitlab.com/ve-nt/outfieldr.git \
+        && cd outfieldr \
+        && git submodule init \
+        && git submodule update \
+        && zig build -Drelease-safe \
+        && mkdir /build-outputs \
+        && cp bin/tldr /build-outputs/outfieldr
+
+################################################################################
+
+FROM ubuntu:latest AS benchmark
+
+ENV PAGE="tar"
+ENV LANG="en_US.UTF-8"
+
+WORKDIR /deps
+RUN apt-get update && apt-get install -y wget unzip && rm -rf /var/lib/apt/lists/*
+COPY --from=tldr-c-builder /deps/* ./
+RUN for file in *; do DEBIAN_FRONTEND=noninteractive sh $file; done
+
+WORKDIR /clients
+COPY --from=tealdeer-builder /build-outputs/* ./
+COPY --from=tldr-c-builder /build-outputs/* ./
+COPY --from=haskell-builder /build-outputs/* ./
+RUN wget -qO tldr-bash https://4e4.win/tldr && chmod +x tldr-bash
+COPY --from=node-builder /build-outputs/node /build-outputs/tldr-node ./
+COPY --from=node-builder /build-outputs/node_modules/ ./node_modules/
+COPY --from=zig-builder /build-outputs/* ./
+
+# Update all the individual caches
+RUN bash -c 'mkdir -p /caches/{tealdeer,tldr-c,tldr-hs,fast-tldr,tldr-bash,tldr-node,outfieldr/.local/share}' \
+        && TEALDEER_CACHE_DIR=/caches/tealdeer ./tealdeer -u \
+        && TLDR_CACHE_DIR=/caches/tldr-c ./tldr-c -u \
+        && XDG_DATA_HOME=/caches/tldr-hs ./tldr-hs -u \
+        && XDG_DATA_HOME=/caches/fast-tldr ./fast-tldr -u \
+        && XDG_DATA_HOME=/caches/tldr-bash ./tldr-bash -u \
+        && HOME=/caches/tldr-node ./tldr-node -u \
+        && HOME=/caches/outfieldr ./outfieldr --fetch
```

Comment:
Outfieldr now uses the same flags as defined in the spec, so `--fetch` has been replaced with `-u`.